app/uploaders/configurable_uploader_versions.rb

```diff
@@ -0,0 +1,30 @@
+module ConfigurableUploaderVersions
```

Comment:
I should have noticed this before, but should this and the ItemUploader be Spotlight-namespaced?